src/lib/claimMgr.js

```diff
@@ -7,12 +16,16 @@ class ClaimMgr {
   }
 
   isSecretsSet () {
-    return (this.signerPrivate !== null || this.signerPublic !== null)
+    return ( this.ipfs !== null && (this.signerPrivate !== null || this.signerPublic !== null))
```

Comment:
Should this not be `and` and not `or` for all? I.e. if any one of these variables are not set then this should return false?

Reply:
👍